nodejs/aws-infra/experimental/ec2.ts

```diff
@@ -0,0 +1,146 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+import * as utils from "../utils";
+
+import * as mod from ".";
+
+export type EC2TaskDefinitionArgs = utils.Overwrite<mod.TaskDefinitionArgs, {
```

Comment:
still using an overwrite here.  but this should be converted to an interface.  annying that we'll have to copy over all the properties another time.